airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/stream_reader.py

```diff
@@ -141,7 +141,7 @@ def upload(
         file_size = self.file_size(file)
         # I'm putting this check here so we can remove the safety wheels per connector when ready.
         if file_size > self.FILE_SIZE_LIMIT:
-            message = "File size exceeds the 1 GB limit."
+            message = f"File size exceeds the 1 GB limit. File uri: {file.uri}"
```

Comment:
Should we instead:
* Improve the log in the CDK [here](https://github.com/airbytehq/airbyte-python-cdk/blob/6395265ae8604d307d80efb7a7a40fb58ee4caf6/airbyte_cdk/sources/file_based/file_based_stream_reader.py#L175)
* Update source-sftp-bulk to use the CDK logic